pages/price-feeds/getting-started.mdx

```diff
@@ -0,0 +1,15 @@
+## Getting Started
+
+Integrating Pyth price feeds into your application is quick and easy.
+
+### Build 🧑‍💻
+
```

Comment:
Every section needs to start with a paragraph of text. You can't start a section and immediately jump to a code block or outline -- there needs to be text providing the context. 

The paragraph could be one sentence, like "Developers interested in building on Pyth can reference the following resources:" 

(This guideline is just how English document structure works. You'll never see e.g., an academic paper that doesn't follow this guideline. It is ok if the sentence is somewhat repetitive with the section heading. People don't read the section headings in the same way as sentences, so they expect some level of repetition here.)